pymc3/distributions/continuous.py

```diff
@@ -1661,6 +1662,103 @@ def logcdf(self, value):
         )
 
 
+class AsymmetricLaplace(Continuous):
+    r"""
+    Asymmetric-Laplace log-likelihood.
+
+    The pdf of this distribution is
+
+    .. math::
+        {f(x|\\b,\kappa) =
+            \left({\frac{\\b}{\kappa + 1/\kappa}}\right)\,e^{-(x)\\b\,s\kappa ^{s}}}
```

Comment:
I think this equation should include `mu` parameter as per wiki page.

Reply:
sure, I will add that.
Thanks though